maria-canqui/lesson-1/exercise-contacts/contacts-manager/Contact.java

```diff
@@ -0,0 +1,5 @@
+class Contact {
+    String name;
```

Comment:
According to the goods practices our fields should be private.
So we have to create getters and setters for them